plugins/modules/credential.py

```diff
@@ -30,6 +30,12 @@
     description:
       - Setting this option will change the existing name (lookup via name).
     type: str
+  copy_from:
+    description:
+      - Name of the existing credential to copy.
+      - If set, copies the specified credential.
+      - The new credential will be created with the name given in the C(name) parameter.
```

Comment:
Can you please add  version_added: 3.0.0?

Reply:
We are going to release 2.6, not 3.0. @alinabuzachis I think it is a mistake. No major release planned, this is not a breaking change.